charts/argo-rollouts/values.yaml

```diff
@@ -477,3 +477,11 @@ notifications:
     # trigger.on-purple: |
     #   - send: [my-purple-template]
     #     when: rollout.spec.template.spec.containers[0].image == 'argoproj/rollouts-demo:purple'
+
+  # -- The subscriptions define the subscriptions to the triggers in a general way for all rollouts
+  subscriptions: {}
+    # subscriptions:      
+    #   recipients:
+    #     - slack:<channel>
+    #   triggers:
+    #     - on-rollout-completed
```

Comment:
nits; please add new blank line in the end

Reply:
Done!

Reply:
Are you sure...?
<img width="942" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/d5fc6dec-7506-49e1-a246-70dc826f0de6">